internal/semver/semver.go

```diff
@@ -0,0 +1,44 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package semver
```

Comment:
Can we add a package description for it.

Reply:
Sure, added the description. 
(Other packages under `internal` do not have package description either. Looks like we need a way to keep consistent across PR reviews in the future.)